config/pipelines/pipeline.conf

```diff
@@ -123,6 +123,28 @@ sources=[
     "source": {
       type="com.arpnetworking.metrics.common.sources.CollectdHttpSourceV1"
       name="collectd_http_source"
+    },
+    {
+      type="com.arpnetworking.metrics.common.sources.KafkaSource"
+      name="kafka_source"
+      consumer={
+        type="org.apache.kafka.clients.consumer.Consumer"
+        topics=["topic"]
+        configs={
+          # Set any properties defined at: https://kafka.apache.org/documentation/#consumerconfigs
+          bootstrap.servers="localhost:9092"
+          group.id="group0"
+          client.id="consumer0"
+          key.deserializer="org.apache.kafka.common.serialization.StringDeserializer"
+          value.deserializer="org.apache.kafka.common.serialization.StringDeserializer"
+          auto.offset.reset="earliest"
+        }
+      }
+      parser={
+        type="com.arpnetworking.test.StringParser"
```

Comment:
Can we configure this by default to use the AMP Protobuf parser?

Reply:
Do you mean change the config file to use that parser or in the case that a parser is not specified, use the protobuf parser (in the source java code)?

Reply:
I believe that the code should require a parser since Kafka is a format agnostic source. 

The config you have should use a real parser. I realize this likely means you have to send real data in your tests.